docs/70-jobs.md

```diff
@@ -13,25 +13,25 @@ Creating a Job starts with defining the necessary properties in your `.aztk/job.
 Each Job has one or more applications given as a List in Job.yaml. Applications are defined using the following properties:
 ```yaml
   applications:
-    - name: 
```

Comment:
I thought we decided to leave in white spaces in yaml since it is required.

Reply:
Hhm i guess its the auto formatting that removed those